### PR TITLE
IRIS-4148 Make sure lazy loaded images appear in discussions

### PR DESCRIPTION
**Pull request by garthwebb**
Our approach removes the lazy loading aspects and makes these normal images.

Ticket:

https://wikia-inc.atlassian.net/browse/IRIS-4148

### CODE REVIEW

File: extensions/wikia/Discussions/maintenance/ForumDumper.php [hunk 1]
```diff
@@ -111,6 +111,13 @@ public function getPages() {
 			->WHERE( 'page_namespace' )->IN( self::FORUM_NAMEPSACES )
 			->ORDER_BY( 'idx' )
 			->runLoop( $dbh, function ( &$pages, $row ) use ( &$display_order ) {
+				// A few of these properties were removed and do not appear on some wikis
+				foreach ( [ 'sticky', 'locked', 'protected' ] as $prop ) {
+					if ( !property_exists( $row, $prop ) ) {
+						$row->$prop = 0;
+					}
+				}
+
```
Comment (garthwebb) [inline extensions/wikia/Discussions/maintenance/ForumDumper.php:120]: This code eliminates some noisy warnings.

Comment (mszabo-wikia) [reply extensions/wikia/Discussions/maintenance/ForumDumper.php:120]: AFAIK we performed a site-wide schema update in  [SUS-1690](https://wikia-inc.atlassian.net/browse/SUS-1690), so these columns should no longer appear anywhere (and they were never used)

Comment (garthwebb) [reply extensions/wikia/Discussions/maintenance/ForumDumper.php:120]: @TK-999, thanks for the update!  I've seen some still in DEV so its good to know I don't need to worry about them in production.

**Comment by jsutterfield** [comment]
LGTM

**Pull request merged**